python/phylanx/ast/utils.py

```diff
@@ -94,27 +96,23 @@ def _format(node, level=0):
     return _format(node)
 
 
-
-
 def save_to_file(data, ofn, verbose=False):
     d = os.path.dirname(ofn)
     if d != "":
         os.makedirs(d, exist_ok=True)
 
     with open(ofn, "w") as f:
-        print(data, file = f)
-    if verbose == True:
+        print(data, file=f)
+    if verbose is True:
         print(ofn, "saved")
 
 
 def dump_to_file(data, key, kwargs):
-    if kwargs[key] != False:
-        if kwargs[key] == True:
+    if kwargs[key] is not False:
+        if kwargs[key] is True:
             # generating file name automatically
```

Comment:
Isn't `kwargs[key] is not False` implying `kwargs[key] is True`?

Reply:
kwargs["dump_python_src"] may have 3 kinds of values:
True:  Dumping to output file using auto-generated file name
False: Do not dump
String: Dumping to output file using this user provided "string" as the file name

I will document this behavior besides this procedure.